app/views/forecast_snapshots/show.html.erb

```diff
@@ -16,9 +16,11 @@
   var all_data = [];
 
   <% @forecast_snapshot.history_tsd.get_all_series.each do |series| %>
+  <% @forecast_snapshot.check_restricted(series[:name] + '.' + series[:frequency]) %>
```

Comment:
Since we reuse the series.mnemonic a lot here, we should store it at the top of this each. e.g.,
```
mnemonic = series[:name] + '.' + series[:frequency]
```